src/Redmine/Client.php

```diff
@@ -204,6 +182,94 @@ public function getUrl()
         return $this->url;
     }
 
+    /**
+     * HTTP GETs a json $path.
+     *
+     * @param string $path
```

Comment:
i think we can remove all those docblocks when the method signature forces the input type hint and well as the return type

Reply:
Sure, will do this. :+1: